src/components/menu/Menu.tsx

```diff
@@ -0,0 +1,237 @@
+import React, { ReactElement, ReactNode, SyntheticEvent, useEffect, useRef, useState } from 'react';
+import cx from 'classnames';
+import { events } from '../utils';
+import { getViewport } from '../utils/utils';
+import Box, { pickBoxProps } from '../box';
+import MenuItem from './MenuItem';
+import theme from './theme.css';
+import uiUtilities from '@teamleader/ui-utilities';
+import { BoxProps } from '../box/Box';
+import { GenericComponent } from '../../@types/types';
+
+export type POSITIONS = 'auto' | 'static' | 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
```

Comment:
PP: types should be PascalCase

Reply:
Yeah, good point, will change it so we have consistency 😉. Thanks!